# generated by patch-package 6.4.8 on 2022-09-13 17:19:44
#
# command:
#   npx patch-package @vscode/test-electron
#
# declared package:
#   @vscode/test-electron: ^2.1.5
#
diff --git a/node_modules/@vscode/test-electron/out/runTest.js b/node_modules/@vscode/test-electron/out/runTest.js
index edf5f22..e9f2a22 100644
--- a/node_modules/@vscode/test-electron/out/runTest.js
+++ b/node_modules/@vscode/test-electron/out/runTest.js
@@ -62,7 +62,11 @@ async function innerRunTests(executable, args, testRunnerEnv) {
         const fullEnv = Object.assign({}, process.env, testRunnerEnv);
         const cmd = cp.spawn(executable, args, { env: fullEnv });
         cmd.stdout.on('data', d => process.stdout.write(d));
-        cmd.stderr.on('data', d => process.stderr.write(d));
+        cmd.stderr.on('data', d => {
+			const str = String(d)
+			if (str.match(/(rejected promise)/)) process.stderr.write(d);
+			// if (str.match(/(rejected promise)|(stack trace:)/)) process.stderr.write(d);
+        });
         cmd.on('error', function (data) {
             console.log('Test error: ' + data.toString());
         });
